public/app/core/utils/kbn.ts

```diff
@@ -228,18 +228,10 @@ kbn.slugifyForUrl = str => {
     .replace(/ +/g, '-');
 };
 
+/** deprecated since 6.1, use grafana/ui */
 kbn.stringToJsRegex = str => {
-  if (str[0] !== '/') {
-    return new RegExp('^' + str + '$');
-  }
-
-  const match = str.match(new RegExp('^/(.*?)/(g?i?m?y?)$'));
-
-  if (!match) {
-    throw new Error(`'${str}' is not a valid regular expression.`);
-  }
-
-  return new RegExp(match[1], match[2]);
+  console.warn('Migrate stringToJsRegex to grafana/ui, ');
```

Comment:
Cool! WDYT about introducing `method/functionDeprecationWarning` in `utils` to have one generic message for such cases? I have created sth like that some time ago for props https://github.com/grafana/grafana/blob/master/packages/grafana-ui/src/utils/propDeprecationWarning.ts

Reply:
rather than add a new identical function, I renamed that `deprecationWarning.ts` and used it in both places.

Reply:
If that sounds OK, i think this is ready to merge